tst/testinstall/perm.tst

```diff
@@ -223,7 +223,8 @@ gap> List([1,2,3,4,1000],n->List(permBig, g->n^g));
 # QuoIntPerm
 #
 gap> 0/(1,2);
-Error, Perm. Operations: <point> must be a positive integer (not 0)
+Error, PermQuotient: <point> must be a positive small integer (not the integer\
```

Comment:
The same problem appears here..

Reply:
woops, you are right. Now fixed, with better warnings